index.ts

```diff
@@ -434,6 +436,24 @@ server.setRequestHandler(CallToolRequestSchema, async (request) => {
           content: [{ type: "text", text: JSON.stringify(comment, null, 2) }],
         };
       }
+      case LIST_TASK_COMMENTS_TOOL.name: {
+        const comments = await CommentService.listComments(
+          args.author as string,
+          args.author_id as string,
+          args.ids as string,
+          args.limit as number,
+          args.offset as number,
+          args.parent_id as string | null,
+          args.published_at_after as string,
+          args.published_at_before as string,
+          args.task as string,
+          args.task_id as string,
+          args.text as string
+        );
```

Comment:
This is the only one where we do this, why not just 
```suggestion
        const comments = await CommentService.listComments(args);
```

Reply:
Fixed. Thanks.